.github/ISSUE_TEMPLATE/release-tracking-issue.md

```diff
@@ -31,7 +31,7 @@ Note: remove the backticks (``) from the link.
 -->
 
 - [x] Release `[<RELEASE_NUMBER>](https://github.com/ampproject/amphtml/releases/tag/<RELEASE_NUMBER>)` is cut as a new canary release
-- [ ] Release <RELEASE_NUMBER> pushed to dev channel (<CL submit time>)
+- [ ] Release <RELEASE_NUMBER> pushed to experimental channel (<CL submit time>)
```

Comment:
While you're at it, I think you can replace the word "pushed" with the word "promoted", since that's the verb we use elsewhere. WDYT?

Reply:
Donezo